client/main.cpp

```diff
@@ -238,7 +238,7 @@ static void init_core_client(int argc, char** argv) {
     // NOTE: this must be called BEFORE newer_version_startup_check()
     //
     if (read_vc_config_file()) {
-        msg_printf(NULL, MSG_INFO, "nvc_config.xml not found - using defaults");
+       // msg_printf(NULL, MSG_INFO, "nvc_config.xml not found - using defaults");
     }
```

Comment:
Did you intend to have this commented out? It seems like it should either be enabled, be surrounded by one of the flags for logging or to be rewritten to remove the if block.

Reply:
Yes.  It's a debugging message, printed before cc_config.xml has been read.  I commented it out in case Charlie needs it again at some point.

Asides:
1) the NVC_CONFIG stuff has no comments as far as I can see; it should.
2) why is the NVC initialization done in main.cpp (low level)
   rather than in CLIENT_STATE::init()?